GTFS/GTFSWriter.cs

```diff
@@ -162,23 +173,22 @@ public void Write(T feed, IEnumerable<string> idsToWrite, IEnumerable<IGTFSTarge
             var calendarDatesToWrite = feed.CalendarDates.Where(x => serviceIds.Contains(x.ServiceId)).ToList();
             var levelsToWrite = feed.Levels.ToList();
             var pathwaysToWrite = feed.Pathways.ToList();
-            
 
             // remove null stops
             stopsToWrite = stopsToWrite.Where(x => x != null).ToList();
 
             // order files by id
-            agenciesToWrite = agenciesToWrite.OrderBy(x => x.Id).ToList();
+            agenciesToWrite = agenciesToWrite.OrderBy(x => x.Name ?? x.Id).ToList();
             calendarDatesToWrite = calendarDatesToWrite.OrderBy(x => x.ServiceId).OrderBy(y => y.ExceptionType).OrderBy(z => z.Date).ToList();
             calendarsToWrite = calendarsToWrite.OrderBy(x => x.ServiceId).ToList();
             fareAttributesToWrite = fareAttributesToWrite.OrderBy(x => x.FareId).ToList();
             fareRulesToWrite = fareRulesToWrite.OrderBy(x => x.RouteId).ToList();
             frequenciesToWrite = frequenciesToWrite.OrderBy(x => x.TripId).ToList();
-            routesToWrite = routesToWrite.OrderBy(x => x.Id).ToList();
-            stopsToWrite = stopsToWrite.OrderBy(x => x.Id).ToList();
-            stopTimesToWrite = stopTimesToWrite.OrderBy(x => x.TripId).ToList();
+            routesToWrite = routesToWrite.OrderBy(x => x.ToString()).ToList();
+            stopsToWrite = stopsToWrite.OrderBy(x => x.Name ?? x.Id).ToList();
+            stopTimesToWrite = stopTimesToWrite.OrderBy(x => x.StopSequence).OrderBy(x => x.TripId).ToList();
```

Comment:
Surely you want to order by TripId first and then by stop sequence?

Reply:
nope, this is correct
![image](https://user-images.githubusercontent.com/22648295/74648940-464b5b80-5187-11ea-9bd0-131468a242a6.png)

Reply:
I think both would give the correct output, this just seems a bit strange to me and probably requires a lot more shuffling than doing it the other way round. The intermediate step has all stops with the same sequence next to each other which just seems weird. But it does work.

Reply:
Happy though to complete

Reply:
I think sorting the other way round will give you stop times with sequences 111122223333 instead of 123123123123.